tests/utils/storyFabricator.js

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2018 OCAD University
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling-server/master/LICENSE.txt
```

Comment:
There are a number of places that the github "sjrk-story-telling-server" repo is still referred to. Can you please update? Thanks.